### PR TITLE
Auto-require of bundler

### PR DESCRIPTION
**Pull request by KrauseFx**
Solves a problem on some environments

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -91,6 +91,8 @@ end
 
 task :test_all do
   exceptions = []
+  require 'bundler'
```
Comment (i2amsam) [inline Rakefile:94]: What does this fix?

**Comment by KrauseFx** [comment]
Sometimes you'd get an error that `Bundler` is not defined, I think it depends on the version of Ruby. This will just make sure to automatically include this :+1:

**Comment by i2amsam** [comment]
Any reason to require it here rather than at the top of the `Rakefile`?

**Comment by KrauseFx** [comment]
Because it's only needed for this one rake task. Look a few lines below, we actually call `Bundler.something`

**Comment by i2amsam** [comment]
:+1:

**Pull request merged**